src/devtools/views/Components/KeyValue.css

```diff
@@ -26,3 +26,13 @@
   color: var(--color-dimmer);
   font-style: italic;
 }
+
+.Opener {
+  padding: 0.125rem;
+  margin-left: -1rem;
```

Comment:
Strong preference to avoid negative margins.

Reply:
just out of curiosity, why is that?

Reply:
When things overflow their container, it makes it harder to reason about the layout. Things can also get clipped pretty easily if any ancestors hide overflow.